### PR TITLE
create InitData class for data initialization directly to the databas…

### PR DESCRIPTION
**Pull request by adamveszelinov**
please review it. Thanks

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -38,5 +38,6 @@ dependencies {
     compile group: 'javax.mail', name: 'mail', version: '1.4.7'
     compile group: 'org.jsoup', name: 'jsoup', version: '1.11.3'
     compile group: 'javax.activation', name: 'activation', version: '1.1'
+    compile group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.8'
```
Comment (gergelyszanto) [inline build.gradle]: Why not RestAssured?

File: src/test/java/com/mentoring/framework/utils/InitData.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            StringEntity input = new StringEntity(
+                    "{\"username\":\""+ username +"\"" +
+                    ", \"password\":\""+ password +"\"" +
+                    ", \"email\": \""+ email +"\"}");
+
+            input.setContentType("application/json");
+            postRequest.setEntity(input);
+
+            client.execute(postRequest);
+        } catch (UnsupportedEncodingException e) {
```
Comment (gergelyszanto) [inline src/test/java/com/mentoring/framework/utils/InitData.java]: You can merge these 3 exceptions into one catch with pipes. You don't need to include the exception's name into the custom log message if you're adding the stacktrace anyways.

File: src/test/java/com/mentoring/framework/utils/InitData.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    private InitData() {
+    }
+
+    public static void registerUser(String username, String password, String email) {
+        try {
+            HttpClient client = HttpClientBuilder.create().build();
+
+            HttpPost postRequest = new HttpPost(Config.getApplicationUrl() + "/user");
+
+            StringEntity input = new StringEntity(
```
Comment (gergelyszanto) [inline src/test/java/com/mentoring/framework/utils/InitData.java]: This is where RestAssured would look much more readable and clean. With RestAssured you can also declare the expected response code, so your tests can fail-fast if the rest call fails.

File: src/test/java/com/mentoring/framework/utils/InitData.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+            input.setContentType("application/json");
+            postRequest.setEntity(input);
+
+            client.execute(postRequest);
+        } catch (UnsupportedEncodingException e) {
+            log.error("Error happened during user registration (Unsupported Encoding Exception).", e);
+        } catch (ClientProtocolException e) {
+            log.error("Error happened during user registration (Client Protocol Exception).", e);
+        } catch (IOException e) {
```
Comment (gergelyszanto) [inline src/test/java/com/mentoring/framework/utils/InitData.java]: I don't see what happens with the request after all the exceptions are caught. If you can't post a user registration due to an exception, your test shouldn't be able to proceed.

**Review by gergelyszanto — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (adamveszelinov) [reply build.gradle]: I changed the code to RestAssured

**Review by gergelyszanto — APPROVED** [review]

**Pull request merged**